invenio_previewer/templates/semantic-ui/invenio_previewer/pdfjs.html

```diff
@@ -19,14 +19,94 @@
   <meta name="google" content="notranslate">
   <link rel="resource" type="application/l10n" href="{{ url_for('static', filename='js/pdfjs/web/locale/locale.properties') }}" />
   <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='js/pdfjs/web/pdf_viewer.css') }}" />
-  <script src="{{ url_for('static', filename='js/pdfjs/build/pdf.mjs') }}" type="module"></script>
+  <script src="{{ url_for('static', filename='js/pdfjs/build/pdf.min.mjs') }}" type="module"></script>
   <script src="{{ url_for('static', filename='js/pdfjs/web/pdf_viewer.mjs') }}" type="module"></script>
 {%- endblock %}
 
 {%- block page_body %}
 <input id="pdf-file-uri" type="hidden" value="{{ file.uri }}" />
 
-<div id="viewerContainer">
-  <div id="viewer" class="pdfViewer"></div>
-</div>
+  <div id="outerContainer">
+      <div id="mainContainer">
+        <div class="toolbar">
+          <div id="toolbarContainer">
+            <div id="toolbarViewer" class="toolbarHorizontalGroup">
+
+              <div id="toolbarViewerLeft" class="toolbarHorizontalGroup">
+                <div class="toolbarHorizontalGroup hiddenSmallView">
+                  <button class="toolbarButton" title="Previous Page" type="button" id="previous" tabindex="0" data-l10n-id="pdfjs-previous-button">
+                    <span data-l10n-id="pdfjs-previous-button-label">Previous</span>
+                  </button>
+                  <div class="splitToolbarButtonSeparator"></div>
+                </div>
+                <div class="toolbarHorizontalGroup">
+                  <span class="loadingInput start toolbarHorizontalGroup">
+                    <input type="number" id="pageNumber" class="toolbarField" title="Page" value="1" min="1" tabindex="0" data-l10n-id="pdfjs-page-input" autocomplete="off">
+                  </span>
+                  <span id="numPages" class="toolbarLabel"></span>
+                </div>
+                <div class="toolbarHorizontalGroup hiddenSmallView">
+                  <div class="splitToolbarButtonSeparator"></div>
+                  <button class="toolbarButton" type="button" title="Next Page" id="next" tabindex="0" data-l10n-id="pdfjs-next-button">
+                    <span data-l10n-id="pdfjs-next-button-label">Next</span>
+                  </button>
+                </div>
+              </div>
+
+              <div id="toolbarViewerMiddle" class="toolbarHorizontalGroup">
+                <div class="toolbarHorizontalGroup">
+                  <button id="zoomOutButton" class="toolbarButton" type="button" title="Zoom Out" tabindex="0" data-l10n-id="pdfjs-zoom-out-button">
+                    <span data-l10n-id="pdfjs-zoom-out-button-label">Zoom Out</span>
```

Comment:
Question: Should we mark all strings in this template for localization?

Reply:
See: https://github.com/inveniosoftware/invenio-previewer/pull/211/commits/e751832b40e2995111def5b1f53ec982b916684a#diff-7774f279283077d187f6ff5acab22b240e0a875a6789eb686c0bcd25a665ba0b

Reply:
good point; yes, that should actually be done!
i just forgot to mark them :sweat: